docs/quickstart-eks.md

```diff
@@ -0,0 +1,264 @@
+* [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [BKPR installer](install.md)
+* [kubecfg](https://github.com/ksonnet/kubecfg/releases)
+* [jq](https://stedolan.github.io/jq/)
+
+### DNS requirements
+
+In addition to the requirements listed above, a domain name is also required for setting up Ingress endpoints to services running in the cluster. The specified domain name can be a top-level domain (TLD) or a subdomain. In either case you have to manually [set up the NS records](#step-3-configure-domain-registration-records) for the specified TLD or subdomain so as to delegate DNS resolution queries to an Amazon Route 53 hosted zone created and managed by BKPR.  This is required in order to generate valid TLS certificates.
+
+## Installation and setup
+
+### Step 1: Set up the cluster
+
+In this section, you will deploy an Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster using the `eksctl` tool`.
+
+* Make sure you have a working Amazon CLI environment. Please refer to [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) if in doubt.
+
+* Configure the following environment variables:
+
+  ```bash
+  export BKPR_DNS_ZONE=my-domain.com
+  export AWS_EKS_USER=my-user@example.com
+  export AWS_EKS_CLUSTER=my-eks-cluster
+  export AWS_EKS_K8S_VERSION=1.11
+  ```
+
+  - `BKPR_DNS_ZONE` specifies the DNS suffix for the externally-visible websites and services deployed in the cluster.
+  - `AWS_EKS_USER` specifies the email address used in requests to Let's Encrypt.
+  - `AWS_EKS_CLUSTER` specifies the name of the AKS cluster.
+  - `AWS_EKS_K8S_VERSION` specifies the version of Kubernetes to use for creating the cluster. The [BKPR Kubernetes version support matrix](../README.md#kubernetes-version-support-matrix-for-bkpr-10) lists the base Kubernetes versions supported by BKPR.
+
+* Create the EKS cluster:
+
+  ```bash
+  eksctl create cluster --name=${AWS_EKS_CLUSTER} \
+                        --color=fabulous \
+                        --nodes=3 \
+                        --version=${AWS_EKS_K8S_VERSION}
+  ```
+
+  Provisioning an EKS cluster can take a long time to complete. Please be patient while the request is being processed.
+
+* Configure `kubectl` to use the new cluster:
+
+  ```bash
+  aws eks update-kubeconfig --name=${AWS_EKS_CLUSTER}
+  ```
+
+* Verify that your cluster is up and running:
+
+  ```bash
+  kubectl get nodes
+  ```
+
+### Step 2: Configure a User Pool in AWS Cognito
+
+In order to authenticate users for applications running atop EKS, BKPR requires a User Pool in AWS Cognito to be configured.
+
+If you already have a working User Pool in AWS Cognito that you would like to use for authenticating users, you will need to retrieve its ID in the form `${AWS_REGION}-${USER_POOL_ID}`, and write it down. Using this `Pool Id`, skip at the end of this section.
```

Comment:
Shall we request at this point to export it as AWS_COGNITO_USER_POOL_ID (as it says below). Also, what is the end of this section? move to the next one? something else?

Reply:
Done